services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/DatabaseTest.kt

```diff
@@ -19,10 +20,11 @@ import org.testcontainers.utility.MountableFile
  */
 @Sql("/clear-db.sql")
 @SqlMergeMode(SqlMergeMode.MergeMode.MERGE)
+@Testcontainers
 abstract class DatabaseTest {
```

Comment:
Could DatabaseTest be an interface instead of an abstract class?

Reply:
Looks like the container doesn't get started if it's an interface.

Reply:
The kdoc should be revised however.